### PR TITLE
feat: add project seeding api

### PR DESCRIPTION
**Pull request by Muhameti-centralsoft**
Add documentation for the Project Seeding Api

Question: As `channels` are going to depreciation, should I add more explanation about how they work?

### CODE REVIEW

File: content/reference-docs/server-extensions/project-seeding-api.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+You can define users in two places:
+- In the seeding configuration  under `users`(like in the example above)
+- In the `conf/secrets/local.js` file under `development.users`. This way you can define your own email and password
+
+### Projects
+
+Project configurations can be of three types:
+- Multi-channel
+- Embedded design
+- Reference design
```
Comment (ajwild) [inline content/reference-docs/server-extensions/project-seeding-api.md:123]: Is multi-channel always using reference design, or is multi-channel with embedded design possible?

File: content/reference-docs/server-extensions/project-seeding-api.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+- The keys of `groupMemberships` are the `group` labels,
+- The values of `groupMemberships` are arrays with user `_importHandle`'s
+
+#### Config
+
+[Project Config]({{< ref "/reference-docs/project-config" >}})
+
+#### Secrets
+
+Secret values for usage inside a project. Example usage may be for `api-secrets` of external services.
```
Comment (ajwild) [inline content/reference-docs/server-extensions/project-seeding-api.md:142]:
```suggestion
Secret values for usage inside a project. Example usage may be for API secrets of external services.
```

File: content/reference-docs/server-extensions/project-seeding-api.md (hunk 1 above)
Comment (Muhameti-centralsoft) [reply content/reference-docs/server-extensions/project-seeding-api.md:123]: I'm not sure for that, but I asked Lukas who thinks that only referenced designs are possible

File: content/reference-docs/server-extensions/project-seeding-api.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+      _importHandle: ':clark',
+      firstName: 'Clark',
+      lastName: 'Kent',
+      email: 'clark@livingdocs.io',
+      password: 'Secret-Phrase'
+    }
+  ],
+
+  projects: [
+    // project with multi-channel configuration
```
[the remark this answers is not in the document]
Comment (Muhameti-centralsoft) [reply content/reference-docs/server-extensions/project-seeding-api.md:39]: I don't know neither. @peyerluk do you maybe know that?

[the remark this answers is not in the document]
Comment (ajwild) [reply content/reference-docs/server-extensions/project-seeding-api.md]: Not so important now that the multi-channel is at the bottom of the list :+1:

File: content/reference-docs/server-extensions/project-seeding-api.md (hunk 1 above)
Comment (ajwild) [reply content/reference-docs/server-extensions/project-seeding-api.md]: Lets merge and we can edit it later if needed.

**Review by ajwild — APPROVED** [review]

**Pull request merged**